.github/workflows/tests.yaml

```diff
@@ -46,6 +48,7 @@ jobs:
 
       - name: Install dependencies
         run: |
+          [ "$OPTIONS" = "not tf" ] && gawk -i inplace '!/('$SKIP_DEPENDENCIES')/' requirements-ci.txt
```

Comment:
Could this persist between test runs? E.g., if this is executed before the regular tests (which require TensorFlow), could the `requirements-ci.txt` have been changed due to this?

Reply:
When using the [matrix](https://github.com/XanaduAI/strawberryfields/blob/f6038e060756e6948b3173875e566b22581872c3/.github/workflows/tests.yaml#L25-L33) strategy, every entry defines a new job and each job provisions a different virtual environment. This modification will only apply to the env with `OPTIONS="not tf"` and will not persist to other envs.